### DISCUSSION TITLE
Sites acessiveis para surdos, com conteudo em libras

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#210] [Dicas]
Pessoal voces viram isso aqui?
http://www.handtalk.me/ ?

Tava fazendo uma página da Printi hoje aqui e acessei esse site que tinha esse widget instalado.. e achei demais..

http://startupi.com.br/2012/08/printi-grafica-virtual-conta-com-aporte-de-greenoaks-capital-kai-schoppen-brandsclub-florian-otto-groupon-e-fabrice-grinda-olx/

vocês já participaram de sites com projetos para acessibilidade? Sabem de resultados concretos para o site e para os visitantes que essas tecnologias proporcionaram??

Abraços

**Comment by williamgcd** [comment]
Oi Ruben,

Trabalho a 6 anos em uma agência especializada em acessibilidade. Já
trabalhei em muitos projetos acessíveis.

Para surdos, existe também o weblibras.com.br que é bem legal e tem uma
opção free.

No mundo são quase 1 bilhão de pessoas com deficiência e a partir de junho
de 2015 existe uma lei que exige que todos os sites sejam acessíveis. (Tem
uma mais antiga orar sites públicos)

Vale muito correr atrás desse tipo de conhecimento. E, precisando de ajuda,
é só gritar. ;)

Em quarta-feira, 6 de julho de 2016, Ruben Marcus notifications@github.com
escreveu:

> Pessoal voces viram isso aqui?
> http://www.handtalk.me/ ?
> 
> Tava fazendo uma página da Printi hoje aqui e acessei esse site que tinha
> esse widget instalado.. e achei demais..
> 
> http://startupi.com.br/2012/08/printi-grafica-virtual-conta-com-aporte-de-greenoaks-capital-kai-schoppen-brandsclub-florian-otto-groupon-e-fabrice-grinda-olx/
> 
> vocês já participaram de sites com projetos para acessibilidade? Sabem de
> resultados concretos para o site e para os visitantes que essas tecnologias
> proporcionaram??
> 
> Abraços
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/frontendbr/forum/issues/210, or mute the thread
> https://github.com/notifications/unsubscribe/AAuV6bxzd7l3kNC4kpvUhxWsik5My-7_ks5qTD4sgaJpZM4JGnJW
> .

## 

## 

William Daflita

**Comment by fititnt** [comment]
**TL;DR: NÃO instale plugin do handtalk em seus sites**

---

Todos os meus projetos levam em conta acessibilidade, e eu ia responder que iria pesquisar mais a respeito antes de dar uma opinião se realmente vale a pena incluir isso ou não (por exemplo, as pessoas usarem isso como widget no nagegador), mas dai vi isso aqui ao criar uma conta no site:

![handtalk cobra por acesso, tabela de preços](https://cloud.githubusercontent.com/assets/812299/19095841/b8197122-8a6e-11e6-88ef-dab1ad285af0.png)

Ou seja, sem nem mesmo entrar no mérito de instalar um plugin nisso no seu site, que aparentemente vai exibir propaganda, fora toda questão de perigo de sofrer um XSS por incluir JS de um servidor que você não tem controle, eu posso afirmar que **não, não instale handtalk em qualquer projeto**. Não é código aberto, é pago, e ainda tem perigos de segurança.

**Comment by eduardojmatos** [comment]
@fititnt foda, mas concordo contigo :(

**Comment by felipefialho** [comment]
@fititnt Não sabia dessa.

**Comment by ThadeuLuz** [comment]
**Aviso**: sou um dos fundadores da Hand Talk

Acho que devemos fazer um trabalho melhor na comunicação sobre como funciona a propaganda. Peço desculpas por não termos feito isso inicialmente.

A 'propaganda' é só uma imagem com um link para uma página sempre em algum domínio nosso (handtalk.me ou amigodosurdo.com), mostrada enquanto o plugin abre. Não tem javascript de terceiros pra rede de afiliados ou coisa do tipo. Até as do Uber que vc vê hoje, são links para posts no nosso blog. O único JavaScript de terceiros que usamos é o gerado pelo Unity 3D, que é nosso engine pra webGL.

[Nesse link](http://suporte.handtalk.me/hc/pt-br/articles/223125927-Considera%C3%A7%C3%B5es-sobre-a-seguran%C3%A7a-do-Tradutor-de-Sites) tem mais informações sobre segurança. Somos um time pequeno, mas estamos considerando a possibilidade de abrir o código fonte do tradutor de sites.

**Comment by fititnt** [comment]
Ok. Como o assunto é mais complexo, eu vou parar e descrever mais os meus motivos pessoais de porque eu não recomendaria a inclusão do script em páginas, da forma como ele está na data de hoje. 

Isso vai dar um pouco de trabalho.

**Comment by fititnt** [comment]
Pronto. Feedback completo em https://github.com/fititnt/handtalk-feedback

Aqui o **TL;DR:**
- **Você NÃO fere lei por não incluir este script proprietário**, ao contrário
  do que o site poderia deixar subentendido
- **Deixa o site mais lento**: 208KB de arquivos para uma uma funcionalidade que
  raramente vai ser usada. Note: não tenho críticas a 19MB necessário quando a solução
  é usada
- **Requer JavaScript e extrema confiança nos sites que os usam**: o uso do
  script permite controle absoluto sobre o que ocorre na página para TODOS os
  usuários (não apenas surdos).
- **Os termos de uso abrem margem para uso para fins comerciais se a empresa for
  comprada**, o combo muitos sites usando a versão gratuita e o poder de
  rastreamento tornaria esse projeto altamente interessante para análise de
  concorrência.
- **Uma forma de mitigar essas críticas é carregar script dos servidores do
  HandTalk apenas quando for efetivamente necessário**, melhora performance de
  sites e protege privacidade de usuários não surdos

**Comment by ConselheiroCristao** [comment]
Certamente é uma ótima funcionalidade, vou estar vendo uma forma de implementar isso no site Conselheiro Cristão!
